src/transformState.js

```diff
@@ -46,7 +46,29 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const transform of transforms) {
+    const operation = transform.operation;
```

Comment:
```suggestion
    const { operation } = transform;
```
https://learn.javascript.ru/destructuring